sql/moz-fx-data-shared-prod/mozilla_vpn_derived/waitlist_funnel_v1/query.sql

```diff
@@ -0,0 +1,102 @@
+WITH send_ids AS (
+  SELECT
+    SendID,
+    ARRAY_AGG(DISTINCT URL) AS URLs,
+  FROM
+    `mozilla-cdp-prod`.sfmc.clicks
+  WHERE
+    URL LIKE "%://vpn.mozilla.org%"
```

Comment:
If we linked to vpn.mozilla.org from a non-waitlist email (e.g. a newsletter), then won't this query include those emails as being waitlist emails?

Reply:
Looking throug `sfmc.sendjobs`, can we identify VPN waitlist emails based on the FromName, FromEmail, EmailName, or the `ListID` in `sfmc.sent`?